model_compression_toolkit/qat/keras/quantization_facade.py

```diff
@@ -56,19 +56,27 @@
     DEFAULT_KERAS_TPC = get_target_platform_capabilities(TENSORFLOW, DEFAULT_TP_MODEL)
 
 
-    def qat_wrapper(n: common.BaseNode, layer: Layer, qat_config):
+    def qat_wrapper(n: common.BaseNode,
+                    layer: Layer,
+                    qat_config,
+                    wrap_with_activation_quantizers: bool=True):
         """
         A function which takes a computational graph node and a keras layer and perform the quantization wrapping
         Args:
             n: A node of mct graph.
-            layer: A keras layer
+            layer: A keras layer.
+            wrap_with_activation_quantizers: Whether to use the wrapper for the activation quantizer or not
+
 
         Returns: Wrapped layer
 
         """
         if _is_qat_applicable(n, DEFAULT_KERAS_INFO):
             weights_quantizers, activation_quantizers = quantization_builder(n, qat_config, DEFAULT_KERAS_INFO)
-            return qi.KerasQuantizationWrapper(layer, weights_quantizers, activation_quantizers)
+            if wrap_with_activation_quantizers:
```

Comment:
Update this according to what you decide regarding the terminology of "wrapper"/"holder".